addons/sys_prc77/radio/fnc_getExternalAudioPosition.sqf

```diff
@@ -20,4 +20,20 @@
  * Public: No
  */
 
-[0,0,0];
+params ["_radioId", "_event", "_eventData", "_radioData"];
+
+private _obj = [_radioId] call EFUNC(sys_radio,getRadioObject);
+private _pos = getPosASL _obj;
+
+private _rackId = [_radioId] call EFUNC(sys_rack,getRackFromRadio);
```

Comment:
This is all same for all radios, with just 2 cases based on handheld or rack radio. An outside function with arguments for that would probably be a good idea.